_posts/2025-03-31-mmaler-blogpost-1-intro.adoc

```diff
@@ -61,11 +53,44 @@ Unlike traditional reactive-only frameworks, Quarkus enables developers to combi
 This is particularly beneficial for systems requiring high throughput and low latency, ensuring that applications remain robust under heavy load.
 Quarkus's reactive model makes it ideal for event-driven architectures and microservices.
 
+* A basic example of reactive messaging in Quarkus:
++
+[source, java]
+----
+@ApplicationScoped
+public class PriceConverter {
+
+    @Incoming("prices")
+    @Outgoing("converted-prices")
+    public double convert(double priceInEuro) {
+        return priceInEuro * 1.1;
+    }
+}
+----
++
+In this example, prices are received from one channel (`prices`), converted, and sent to another channel (`converted-prices`).
+This pattern supports high-throughput, event-driven processing with clean and efficient logic.
+
+* An example of a reactive HTTP endpoint using reactive routes in Quarkus:
++
+[source,java]
+----
+@ApplicationScoped
+public class GreetingRoute {
+
+    @Route(path = "/hello", methods = HttpMethod.GET)
+    public Uni<String> hello() {
+        return Uni.createFrom().item("Hello from reactive route!");
+    }
+}
+----
++
+This route handles HTTP GET requests reactively using `Uni` from Mutiny, making it easy to build non-blocking, low-latency APIs.
 
 == Which of your current development pains could Quarkus solve effortlessly?
```

Comment:
I'd remove the "effortlessly" part. It's anthropomorphism, and also a very subjective term, which might reek of overpromising...